docs/fedora-installation-guide.md

```diff
@@ -0,0 +1,53 @@
+# Installing Clear Containers 3.0 Alpha on Fedora
+
+Clear Containers **3.0-alpha** is available for Fedora\* versions **24** and **25**.
+
+This step is only required in case Docker is not installed on the system.
+1. Install latest version of Docker with the following commands:
+
+```
+$ sudo dnf -y install dnf-plugins-core
+$ sudo dnf config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo
+$ sudo dnf makecache fast
+$ sudo dnf install docker-ce
+```
+
+For more information on installing Docker please refer to the
+[Docker Guide](https://docs.docker.com/engine/installation/linux/fedora)
+
+2. Install the Clear Containers 3.0 components with the following commands:
+
+```
+$ source /etc/os-release
+$ sudo -E VERSION_ID=$VERSION_ID dnf config-manager --add-repo \
+http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3-staging/Fedora\_$VERSION_ID/home:clearcontainers:clear-containers-3-staging.repo
+$ sudo dnf install cc-runtime cc-proxy cc-shim virtcontainers-pause
+```
+
+3.  Configure docker to use Clear Containers by default with the following commands:
+
+```
+$ sudo mkdir -p /etc/systemd/system/docker.service.d/
+$ cat << EOF | sudo tee /etc/systemd/system/docker.service.d/clr-containers.conf
+[Service]
+ExecStart=
+ExecStart=/usr/bin/dockerd -D --add-runtime cc3=/usr/bin/cc-runtime --default-runtime=cc3
+EOF
+```
+
+4. Restart the docker and Clear Containers systemd services with the following commands:
```

Comment:
4. Restart the Docker and Clear Containers systemd services with the following commands: